_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,324 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that functionality that can be operated by device motion, responding to the motion can be disabled to prevent accidental actuation.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] that has [functionality][] that can be operated by [device motion][].
+
+## Expectation
+
+For the test target, [device motion][] [can be disabled][] and the control for disabling it must be a [user interface component][] and it should be [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
```

Comment:
Also, @WilcoFiers is pointing out that "ui component" it ambiguous…
https://github.com/act-rules/act-rules.github.io/pull/421#discussion_r390893467